bin/cron/build_contact_rollups

```diff
@@ -3,11 +3,43 @@
 require_relative('../../dashboard/config/environment')
 require 'cdo/contact_rollups'
 require 'cdo/contact_rollups_validation'
+require 'cdo/pardot'
+require 'cdo/chat_client'
 require 'cdo/only_one'
 
+CHAT_CHANNEL = "cron daily".freeze
+
 def main
+  time_start = Time.now
+
+  # Build new daily table of contact rollups
```

Comment:
Though this may partially be a matter of preference, I find none of the comments in this file worth keeping.

From our [style guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages): "Prefer extracting descriptively-named methods and variables over introducing redundant comments." Your method names are descriptive, don't introduce redundant comments.